executor/internal/vecgroupchecker/vecgroupchecker_test.go

```diff
@@ -0,0 +1,272 @@
+// Copyright 2023 PingCAP, Inc.
```

Comment:
ditto, rename to `vec_group_checker_test.go` is better.

Reply:
updated